entities/entities/money_printer/shared.lua

```diff
@@ -1,8 +1,21 @@
+--Static Vars
 ENT.Type = "anim"
 ENT.Base = "base_gmodentity"
 ENT.PrintName = "Money Printer"
-ENT.Author = "Render Case and philxyz"
+ENT.Author = "DarkRP Developers"
 ENT.Spawnable = false
+ENT.sparking = false
+ENT.IsMoneyPrinter = true
+
+--Vars
+ENT.MoneyCount = GAMEMODE.Config.mprintamount
```

Comment:
Like @Bo98 's situation with the common lab, I think these variables will be set before the DarkRPMod's config files are run.